docs/observability/bigquery-agent-analytics.md

```diff
@@ -0,0 +1,1105 @@
+# BigQuery Agent Analytics Plugin (v2)
+
+<div class="language-support-tag">
+  <span class="lst-supported">Supported in ADK</span><span class="lst-python">Python v1.21.0+</span><span class="lst-preview">Preview</span>
+</div>
+
+!!! important "Version Requirement"
+    To use the V2 features described in this document (including Multimodal Logging and GCS Offloading), please update the BigQuery Agent Analytics Plugin to version **1.21.0 or later**.
```

Comment:
This should probably reference ADK 1.21.0 as opposed to a plugin version here and throughout the page. Is the plugin versioned as v2 separate from the adk-python version?